python/tvm/relay/frontend/pytorch.py

```diff
@@ -0,0 +1,1138 @@
+    'aten::relu_'                           : _relu(),
+    'aten::adaptive_avg_pool2d'             : _adaptive_avg_2d(),
+    'aten::adaptive_max_pool2d'             : _adaptive_max_2d(),
+    'aten::max_pool2d'                      : _maxpool_2d(),
+    'aten::max_pool2d_with_indices'         : _maxpool_2d(),
+    'aten::hardtanh'                        : _hardtanh(),
+    'aten::hardtanh_'                       : _hardtanh(),
+    'aten::_convolution'                    : _convolution(),
+    'aten::softmax'                         : _softmax(),
+    'aten::threshold'                       : _threshold(),
+    'aten::threshold_'                      : _threshold(),
+    'aten::contiguous'                      : _contiguous(),
+    'aten::batch_norm'                      : _batch_norm(),
+    'aten::transpose'                       : _transpose(),
+    'aten::transpose_'                      : _transpose(),
+    'aten::t'                               : _transpose(),
+    'aten::flatten'                         : _flatten(),
+    'aten::addmm'                           : _dense(),
+    'aten::size'                            : _size(),
+    'aten::view'                            : _view(),
+    'aten::clone'                           : _clone(),
+    'aten::log_softmax'                     : _log_softmax(),
+    'aten::sigmoid'                         : _sigmoid(),
+    'aten::avg_pool2d'                      : _avg_pool2d(),
+    'aten::dropout'                         : _dropout(),
+    'aten::dropout_'                        : _dropout(),
+    'aten::mean'                            : _mean(),
+    'aten::chunk'                           : _chunk(),
+    'aten::matmul'                          : _matmul(),
+    'aten::expand'                          : _expand(),
+    'aten::Int'                             : _int(),
+    'prim::NumToTensor'                     : _numtotensor(),
+    'prim::ListUnpack'                      : _listunpack(),
+    'aten::constant_pad_nd'                 : _pad(),
+    'aten::permute'                         : _transpose(),
+    'aten::sum'                             : _reduce('sum'),
+    'aten::prod'                            : _reduce('prod'),
+    'aten::sqrt'                            : _sqrt()
+}
+
+# Internal graph for parsing
+
+class Graph(object):
+    """ A helper class for handling relay graph copying from PyTorch trace. """
+
+    def __init__(self, trace, input_shapes, input_types):
+
+        self._trace = trace
+        self._inputs_r = {}
+        self._params = {}
+        self._param_tensors = {}
+        self._consts = {}
+        self._ops = {}
+        self._op_inputs_r = {}
+        self._op_inputs_types = {}
+        self._op_inputs_otypes = {}
+        self._input_shapes = input_shapes if input_shapes else {}
+        self._input_types = input_types if input_types else {}
+        self._fn_param = []
+        self._relay_map = {}
```

Comment:
remove _relay_map here and use "outputs" inside from_torch. see my comment at L903.